resin_calculator/artifact_calc_test.py

```diff
@@ -12,4 +12,24 @@ def test_validate_rarity():
     assert artifact_calc.validate_rarity("3") == True
     assert artifact_calc.validate_rarity("2") == False
     assert artifact_calc.validate_rarity("1") == False
-    assert artifact_calc.validate_rarity("6") == False
\ No newline at end of file
+    assert artifact_calc.validate_rarity("6") == False
+
+def test_validate_set_selection_five():
+    valid_sets = ["gladiator's", "wanderer's troupe", "viridescent venerer", "thundering fury", "thundersoother"
+                  "crimson witch of flames", "lavawalker", "archaic petra", "retracing bolide", "maiden beloved"
+                  "noblesse oblige", "bloodstained chivalry"]
```

Comment:
This copying of data can go away once you move determining the valid set list into the validation function.

Reply:
Moved to validation function, redundancy removed.